great_expectations/data_context/store/ge_cloud_store_backend.py

```diff
@@ -176,7 +176,7 @@ class GeCloudStoreBackend(StoreBackend, metaclass=ABCMeta):
         }
     )
 
-    DEFAULT_BASE_URL = "https://app.greatexpectations.io/"
+    DEFAULT_BASE_URL = "https://api.greatexpectations.io/"
```

Comment:
we could reference this in our tests to avoid duplication

Reply:
Yes, absolutely -- I just did a replace all, but will try to clean up.